src/selection.js

```diff
@@ -338,4 +343,9 @@ export default class Selection extends Cursor {
     this.range = content.deleteCharacter(this.host, this.range, character)
     this.setSelection()
   }
+
+  isPlainTextBlock () {
+    const {plainText = false} = block.getState(this.host) ?? {}
```

Comment:
It might be easier if we do the normalisation of the state in the `getState()` method. At the moment we don't need a generic state object so we can say it to be an object with a `plainText` property.
And that way we don't need to do the normalisation everywhere the `getState()` method is called.